### PR TITLE
New Parts for machines

### PR DESCRIPTION
**Pull request by rogueshadowwalker**
Added additional parts for machines including a new chassis and a new head with a few extra heads to change out on the first set.

### CODE REVIEW

File: html/changelogs/rogueshadowwalker-PR- 315.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - mparts: "Added additional parts for machines including a new chassis and a new head with a few extra heads to change out on the first set."
```
Comment (BurpleBineapple) [inline html/changelogs/rogueshadowwalker-PR- 315.yml]:
```suggestion
  - rscadd: "Added additional parts for machines including a new chassis and a new head with a few extra heads to change out on the first set."
```
mparts isn't a valid changelog prefix.

**Review by BurpleBineapple — CHANGES_REQUESTED** [review]

File: html/changelogs/rogueshadowwalker-PR- 315.yml (hunk 1 above)
Comment (rogueshadowwalker) [reply html/changelogs/rogueshadowwalker-PR- 315.yml]: tell that to vrazzel, atleast they showed me how to make a PR that asked half the devs to show me how too.

**Comment by BurpleBineapple** [comment]
Note to reviewer, check if the chassis with "skintone = null" work as intended.

**Review by io-util — APPROVED** [review]
Says it was tested, looks fine.

**Pull request merged**